### PR TITLE
v1 standards

### PR DESCRIPTION
**Pull request by Ben-Rey**

### CODE REVIEW

**Comment by Ben-Rey** [comment]
Maybe a little light at the moment. Do you think we need to explain certain parts of the code for beginners?
Other blockchains have also added a section on risks and regulation.

File: docs/build/smart-contract/standards.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- Gas
+
+- Rolls
+
+Learn more about [Massa Units Standard](https://github.com/massalabs/massa-standards/blob/main/units.md)
+
+
+
+*Last updated on July 5, 2023 by Author*
```
Comment (qdrn) [inline docs/build/smart-contract/standards.md:39]: This is done automatically on the generated website.

File: docs/build/smart-contract/standards.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+id: standards
+sidebar_label: Standards
+---
+
+# Massa Standard
+
+In the blockchain ecosystem, standards serve as crucial pillars fostering interoperability and seamless interaction across diverse platforms and applications. This page offers a comprehensive overview of the standards—including token, contract, unit definitions.
```
Comment (qdrn) [inline docs/build/smart-contract/standards.md:8]: I would probably say something along the line of:
"If you'd like to jumpstart your project, from tokens, to NFTs or wallet creation, massa-standard is the place to start as it includes reference implementations of the most common smart contracts". It's probably not good enough as a wrote it fastly.

My point is that most of the developers are probably looking for reference implementations of most common contracts and I don't think it's clear enough that we provide that from what you wrote.

**Comment by qdrn** [comment]
> Other blockchains have also added a section on risks and regulation.

What do you mean ? Can you share a link ?

> Do you think we need to explain certain parts of the code for beginners? 

I would create tutorials on how to create a FT and NFT and cross-reference from here.

File: docs/build/smart-contract/standards.md (hunk 2 above)
Comment (Ben-Rey) [reply docs/build/smart-contract/standards.md:8]: Sure ! I'll do that

**Comment by Ben-Rey** [comment]
> > Other blockchains have also added a section on risks and regulation.
> 
> What do you mean ? Can you share a link ?
>
>>https://opentezos.com/defi/token-standards#:~:text=A%20token%20standard%20is%20an,consistent%20record%20of%20those%20transfers.
> 
> > Do you think we need to explain certain parts of the code for beginners?
> 
> I would create tutorials on how to create a FT and NFT and cross-reference from here.
>
> 👍

**Comment by qdrn** [comment]
> > > > Other blockchains have also added a section on risks and regulation.
> > > 
> > > 
> > > What do you mean ? Can you share a link ?
> > 
> > 
> > https://opentezos.com/defi/token-standards#risks
> > > > Do you think we need to explain certain parts of the code for beginners?
> > > 
> > > 
> > > I would create tutorials on how to create a FT and NFT and cross-reference from here.
> > 
> > 
> > 👍

It could be useful in general but it's too early given the state of the documentation. Maybe adding a small paragraph that explains why it's important to follow a standard is useful tho? Similar to tezos but much lighter I'd say.

**Comment by Ben-Rey** [comment]
> > Other blockchains have also added a section on risks and regulation.
> 
> What do you mean ? Can you share a link ?
> 
> > Do you think we need to explain certain parts of the code for beginners?
> 
> I would create tutorials on how to create a FT and NFT and cross-reference from here.

If you agree, we'll create these tutorials in a future issue, as the standards are being rewritten with protobuf.

**Comment by qdrn** [comment]
> If you agree, we'll create these tutorials in a future issue, as the standards are being rewritten with protobuf.

Yes, of course!

**Review by qdrn — COMMENTED** [review]
I would put standards under its own section under developer tools. WDYT ?

LGTM otherwise!

**Comment by Ben-Rey** [comment]
> developer

Cool, I moved it in the tool section

**Review by qdrn — APPROVED** [review]

**Pull request merged**